TriangleTestNG/main/java/triangle/Triangle.java

```diff
@@ -0,0 +1,80 @@
+package triangle;
+
+/**
+ * Create new Triangle and definite kind of triangle
+ *
+ * @author Irina Bogatko
+ * @version 2.0
+ * @since 04-10-2016
+ */
+public class Triangle {
+    private final int EQUILATERAL = 1;
+    private final int ISOSCELES = 2;
+    private final int ORDINARY = 3;
+    //first side of the triangle
+    private double a;
+    //second side of the triangle
+    private double b;
+    //third side of the triangle
+    private double c;
+
+    /**
+     * Constructor create new Triangle
+     *
+     * @param a first side of the triangle
+     * @param b second side of the triangle
+     * @param c third side of the triangle
+     */
+    public Triangle(double a, double b, double c) {
+        this.a = a;
+        this.b = b;
+        this.c = c;
+    }
+
+    /**
+     * @return first side of the triangle
+     */
+    public double getA() {
+        return a;
+    }
+
+    /**
+     * @return second side of the triangle
+     */
+    public double getB() {
+        return b;
+    }
+
+    /**
+     * @return third side of the triangle
+     */
+    public double getC() {
+        return c;
+    }
+
+    /**
+     * Determining whether the triangle equilateral, isosceles or the usual
+     *
+     * @return kind of the triangle
+     * result = 1, triangle is
+     * result = 2, triangle is
+     * result = 3, triangle is
+     */
+    public int kindOfTriangle() throws Exception {

```

Comment:
What if somebody calls this method without validation? Will it still function well? I would say it must call validation inside